omod/src/main/webapp/resources/scripts/navigator/navigatorModels.js

```diff
@@ -643,15 +642,16 @@ ConfirmationSectionModel.prototype.select = function() {
                 if (!question.isDisabled() && question.showInConfirmation()) {
 
                     // question title is header, line per labeled field
+
                     if (question.multiLineInConfirmation()) {
                         summaryDiv.append("<h3>" + question.title().text() + "</h3>");
                         var displayLines = [];
                         _.each(question.fields, function (field) {
                             if (!field.isDisabled() && field.displayValue()) {
-                                if (field.label) {
```

Comment:
What is this supposed to accomplish?

Reply:
All fields that have a white-space label will be considered to have no label at all. Example:
`
Old: Labels [Hello,      ]
    Hello: World
         : World
New:
    Hello: World, World
 `